store/src/java/com/zimbra/qa/unittest/TestRemoteImapShared.java

```diff
@@ -37,38 +42,42 @@ public void tearDown() throws ServiceException, DocumentException, ConfigExcepti
     }
 
     @Override
-    @Ignore ("failing on remote imap for now")
-    public void testMultiappendNoLiteralPlus() throws Exception {
-
+    protected ZMailbox getImapZMailbox() throws Exception {
+        // return the ZMailbox instance used by the imap listener
+        ZMailbox mbox = TestUtil.getZMailbox(USER);
+        ImapServerListener listener = ImapServerListenerPool.getInstance().get(mbox);
+        Set<ImapRemoteSession> sessions = listener.getListeners(mbox.getAccountId(), 2);
+        ImapRemoteSession session = sessions.iterator().next();
+        ZMailbox zmbox = (ZMailbox) session.getMailbox();
+        return zmbox;
     }
 
     @Override
     @Ignore ("failing on remote imap for now")
-    public void testMultiappend() throws Exception {
+    public void testCatenateUrl() throws Exception {
 
     }
 
     @Override
     @Ignore ("failing on remote imap for now")
-    public void testCatenateUrl() throws Exception {
+    public void testCatenateSimpleNoLiteralPlus() throws Exception {
 
     }
 
     @Override
     @Ignore ("failing on remote imap for now")
-    public void testCatenateSimpleNoLiteralPlus() throws Exception {
+    public void testCatenateSimple() throws Exception {
 
     }
 
     @Override
-    @Ignore ("failing on remote imap for now")
-    public void testCatenateSimple() throws Exception {
+    public void testStoreTagsDirty() throws Exception {
 
     }
 
     @Override
     @Ignore ("failing on remote imap for now")
-    public void testAppendTags() throws Exception {
+    public void testStoreInvalidSystemFlag() throws Exception {
```

Comment:
this comment by @codacy-bot seems ignorable. Comments inside "Ignore" explain why this method is empty.